leftwm-core/src/utils/command_pipe.rs

```diff
@@ -134,7 +134,10 @@ fn build_send_window_to_tag(raw: &str) -> Result<Command, Box<dyn std::error::Er
     let headless = without_head(raw, "SendWindowToTag ");
     let parts: Vec<&str> = headless.split(' ').collect();
     let tag_id: TagId = parts.get(0).ok_or("missing argument tag_id")?.parse()?;
-    Ok(Command::SendWindowToTag(tag_id))
+    Ok(Command::SendWindowToTag {
+        window: None,
+        tag: tag_id,
```

Comment:
I guess this should get some error-handling similar to `MoveToTag`, but for both `window` and `tag` as well, right?